gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeExtension.kt

```diff
@@ -41,5 +43,26 @@ abstract class ComposeExtension @Inject constructor(
      */
     val kotlinCompilerPluginArgs: ListProperty<String> = objects.listProperty(String::class.java)
 
+    /**
+     * A set of kotlin platform types to which the Compose Compiler plugin will be applied.
```

Comment:
```suggestion
     * A set of kotlin platform types to which the Compose plugin will be applied.
```

This probably is useful for other Compose features, like disabling code generation for resources.